sdk/monitor/azure-monitor-opentelemetry-exporter/tests/trace/test_trace.py

```diff
@@ -760,8 +778,57 @@ def test_span_to_envelope_client_gen_ai(self):
         self.assertEqual(envelope.data.base_data.result_code, "0")
 
         self.assertEqual(envelope.data.base_type, "RemoteDependencyData")
-        self.assertEqual(envelope.data.base_data.type, "az.ai.inference")
+        self.assertEqual(envelope.data.base_data.type, "N/A")
```

Comment:
@lzchen once gen_ai mappings is set at the top, this test case will be returning "N/A" instead of gen_ai. Just to confirm, it is what we are expecting.